### ISSUE TITLE
Lots of bombed planets look wrong

### ISSUE THREAD

**Issue by ccmaster001** [sirgwain/craig-stars#348]
![image](https://github.com/user-attachments/assets/72535de3-279f-4db4-ad50-2f9d27092933)

![image](https://github.com/user-attachments/assets/1f2728d2-40d4-4230-8723-0007dd7a4cd8)

**Comment by sirgwain** [comment]
What looks wrong about it?

**Comment by ccmaster001** [comment]
well it has 0 People !!!! Shows 9800 in the other overview and its marked as colonized not empty

**Comment by sirgwain** [comment]
Ah, you killed off all colonists but your bombers don't have scanners so they didn't update.

All ships have a non-pen scanner range 0. I need to update the scan logic to detect ownership on non-pen scans, while in orbit. That way you can know if a planet was emptied or not, but not learn anything else about it.

**Comment by ccmaster001** [comment]
NO!!! It is my Planet ! I got Bombed. It has not updated correct. 
1. is Correct McBride 0 Kolonists
2. is incorrect 9800 kolonists
3. the ownership of the planets is still there.
4. I think the scan gets not updated as i have there no ships and no planet any more, there has to be a check in the logic for 0 Kolonists

**Comment by sirgwain** [comment]
Oh! You lost the planet but still think you own it. Got it.

**Comment by ccmaster001** [comment]
the game thinks i own it and so i have to click a lot of planets to check
they dont exist any more

Am Mo., 19. Aug. 2024 um 18:51 Uhr schrieb sirgwain <
***@***.***>:
